gulpfile.js

```diff
@@ -1,33 +1,31 @@
-var gulp = require('gulp'),
-  gutil = require('gulp-util'),
-  bower = require('bower'),
-  concat = require('gulp-concat'),
-  rename = require('gulp-rename'),
-  jshint = require('gulp-jshint'),
-  stylish = require('jshint-stylish'),
-  shell = require('gulp-shell'),
-  clean = require('gulp-clean'),
-  ngAnnotate = require('gulp-ng-annotate'),
-  usemin = require('gulp-usemin'),
-  uglify = require('gulp-uglify'),
-  minifyCss = require('gulp-minify-css'),
-  runSequence = require('run-sequence'),
-  webserver = require('gulp-webserver'),
-  fs = require('fs'),
-  paths = {
-    src: 'src'
-  };
+var gulp        = require('gulp');
+var gutil       = require('gulp-util');
+var bower       = require('bower');
+var concat      = require('gulp-concat');
+var rename      = require('gulp-rename');
+var jshint      = require('gulp-jshint');
+var stylish     = require('jshint-stylish');
+var shell       = require('gulp-shell');
+var clean       = require('gulp-clean');
+var ngAnnotate  = require('gulp-ng-annotate');
+var usemin      = require('gulp-usemin');
+var uglify      = require('gulp-uglify');
+var minifyCss   = require('gulp-minify-css');
+var runSequence = require('run-sequence');
+var webserver   = require('gulp-webserver');
+var fs          = require('fs');
+var paths       = { src: 'src' };
```

Comment:
Other stuff was changed in there when we were testing stuff. For me that is easier to read and the other stuff got removed.